nailed-it/New Group/PolishLibraryViewController.swift

```diff
@@ -92,6 +93,20 @@ class PolishLibraryViewController: UIViewController, UICollectionViewDelegate, U
 //                self.show(tryItOnViewController, sender: self)
             }
             actionSheetController.addAction(tryItOnAction)
+            let findThisColor = UIAlertAction(title: "Find \(color!.displayName!) Online", style: .default) { action -> Void in
```

Comment:
is there any way you can move this into a separate function?

Reply:
like all of the stuff for this action

Reply:
i would also do the if color!.brand! != "My Color" check before definining the alert action, and then add it within that same if block. then you don't need to check the same flag afterward to add it.